synnax/pkg/distribution/transport/grpc/framer/transport.go

```diff
@@ -86,22 +101,22 @@ func New(pool *fgrpc.Pool) Transport {
 				ClientFunc: func(
 					ctx context.Context,
 					conn grpc.ClientConnInterface,
-				) (fgrpc.GRPCClientStream[*framerv1.WriterRequest, *framerv1.WriterResponse], error) {
-					return framerv1.NewWriterServiceClient(conn).Write(ctx)
+				) (fgrpc.GRPCClientStream[*tsv1.WriterRequest, *tsv1.WriterResponse], error) {
```

Comment:
We should rename this back to framer. `tsv1` doesn't make a ton of sense in this context

Reply:
I think the idea was that this package also imports the `framer` package. I will rename it to `framerv1`.